extensions/amp-analytics/0.1/vendors.js

```diff
@@ -438,6 +438,35 @@ export const ANALYTICS_CONFIG = {
         '${actionParameter}${parameterSuffix}',
     },
   },
+
+  'shinystat': {
+    'requests': {
+      'ssurl': 'https://amp.shinystat.com/cgi-bin/getcod.cgi',
+      'ssparam': 'AMP=1' +
+                 '&USER=${account}' +
+                 '&AMPPAG=${page}' +
+                 '&AMPHR=${canonicalUrl}' +
+                 '&AMPREFER=${documentReferrer}' +
+                 '&AMPRES=${screenWidth}X${screenHeight}' +
+                 '&AMPCOLOR=${screenColorDepth}' +
+                 '&AMPCID=${clientId(AMP_CID)}' +
+                 '&AMPPID=${pageViewId}' +
+                 '&AMPTTL=${title}' +
+                 '&AMPRQC=${requestCount}' +
+                 '&AMPVIE=${viewer}' +
+                 '&AMPSSIDC=${queryParam(ssidc)}' +
+                 '&${ampp1}=${ampv1}' +
+                 '&${ampp2}=${ampv2}',
+      'pageview': '${ssurl}?${ssparam}',
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true,
```

Comment:
The italian law is quite restrictive on the data we can keep on the logs, so our system has to execute javascript code on the browser in order to perform computations client-side and to store data in the cookies.
We cannot use the "amp-iframe" tag because is does not provide any way to pass the value of document.referrer into the iframe.